test/local/server.js

```diff
@@ -85,16 +84,19 @@ describe('lib/server', () => {
         db = mocks.mockDB({
           devices: [ { id: 'fake device id' } ]
         })
-        instance = server.create(log, error, config, routes, db, translator, Token)
+
+        return server.create(log, error, config, routes, db, translator, Token).then((s) => {
+          instance = s
+        })
       })
 
       it('returned a hapi Server instance', () => {
-        assert.ok(instance instanceof hapi.Server)
+        assert.ok(instance.version)
```

Comment:
Is the test description still accurate with this change? Should we change it to something more appropriate? (or, if the assertion is no longer providing value, let's delete this test entirely)

Reply:
Yeah I think even before this wasn't that useful because everything below expects `instance` to work